.github/workflows/kitchen.vagrant.yml

```
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# vim: ft=yaml
+---
+name: 'Kitchen Vagrant (FreeBSD)'
+'on': ['push', 'pull_request']
+
+env:
+  KITCHEN_LOCAL_YAML: 'kitchen.vagrant.yml'
+
+jobs:
+  test:
+    runs-on: 'macos-10.15'
+    strategy:
+      fail-fast: false
+      matrix:
+        instance:
+          # - prod-server-freebsd-130-master-py3
+          - freebsd-130-master-py3
+          # - prod-server-freebsd-123-master-py3
+          - freebsd-123-master-py3
+          # - prod-server-freebsd-130-3004-0-py3
+          # - prod-server-freebsd-123-3004-0-py3
```

This is the block defined using the `ssf-formula`:

```suggestion
          - default-freebsd-130-master-py3
          - default-freebsd-123-master-py3
          # - default-freebsd-130-3004-0-py3
          # - default-freebsd-123-3004-0-py3
```